src/deepsparse/transformers/pipelines/question_answering.py

```diff
@@ -186,11 +186,19 @@ def output_schema(self) -> Type[BaseModel]:
         """
         return QuestionAnsweringOutput
 
-    # Override engine_forward to account for multiple sets of inputs (multiple spans)
-    def engine_forward(self, inputs: List[List[numpy.ndarray]], **kwargs):
+    def engine_forward(
+        self,
+        engine_inputs: List[List[numpy.ndarray]],
+    ) -> List[List[numpy.ndarray]]:
+        """
+        runs one forward pass for each span of the preprocessed input
+
+        :param engine_inputs: list of multiple inputs to engine forward pass
+        :return: result of each forward pass
+        """
         engine_outputs = []
-        for inps in inputs:
-            engine_outputs.append(self.engine(inps))
+        for inputs in engine_inputs:
+            engine_outputs.append(self.engine(inputs))
 
         return engine_outputs
```

Comment:
nit: Maybe use list comprehension for `engine_outputs`? It will avoid `creation then modification` code smell